public/extended_summary.json

```diff
@@ -0,0 +1,5109 @@
+[
+  {
+    "name": "affinder",
+    "version": "0.2.3",
+    "display_name": "affinder",
+    "summary": "Quickly find the affine matrix mapping one image to another using manual correspondence points annotation",
+    "author": "Juan Nunez-Iglesias",
+    "license": "BSD-3",
+    "home_page": "https://github.com/jni/affinder",
+    "pypi_versions": [
+      "0.1.0",
+      "0.2.0",
+      "0.2.1",
+      "0.2.2",
+      "0.2.3"
+    ],
+    "conda_versions": [
+      "0.2.3",
+      "0.2.2"
+    ]
+  },
+  {
+    "name": "arcos-gui",
+    "version": "0.0.6",
+    "display_name": "napari ARCOS",
+    "summary": "A napari plugin to detect and visualize collective signaling events",
+    "author": "Benjamin Gr\u00e4del",
+    "license": "BSD-3-Clause",
+    "home_page": "https://github.com/bgraedel/arcos-gui",
+    "pypi_versions": [
```

Comment:
@aganders3 could we get both pypi and conda versions sorted in descending order?

Thanks!